### DISCUSSION TITLE
Principle: Contributions should be long-term projects

### DISCUSSION THREAD

**Discussion by jonfreedman** [finos/technical-oversight-committee#24] [General]
We're looking for things that are either demonstrably > v1.0.0 or have a clear path and commitment to get there.

While there's certainly value to a member deciding to contribute a mature project that's not really being modified a lot internally because it's got value for others, but that's probably not the standard contribution.

The contribution process talks about tentative roadmap & the incubation checklist mentions "The Project Team seeks more contributions, as well as wider adoption and feedback; the Project has a good idea of tasks (e.g. "good first issue) for new contributors to work on."

**Comment by ColinEberhardt** [comment]
I agree, this is an important principle. 👍
